doc/examples/denoise_gibbs.py

```diff
@@ -0,0 +1,296 @@
+
+ax.flat[0].imshow(t1_gibbs, cmap="gray", vmin=100, vmax=400)
+ax.flat[0].annotate('Rings', fontsize=10, xy=(81, 70),
+                    color='red',
+                    xycoords='data', xytext=(30, 0),
+                    textcoords='offset points',
+                    arrowprops=dict(arrowstyle="->",
+                                    color='red'))
+ax.flat[0].annotate('Rings', fontsize=10, xy=(75, 50),
+                    color='red',
+                    xycoords='data', xytext=(30, 0),
+                    textcoords='offset points',
+                    arrowprops=dict(arrowstyle="->",
+                                    color='red'))
+
+ax.flat[1].imshow(t1_unring, cmap="gray", vmin=100, vmax=400)
+ax.flat[1].annotate('WM/GM', fontsize=10, xy=(75, 50),
+                    color='green',
+                    xycoords='data', xytext=(30, 0),
+                    textcoords='offset points',
+                    arrowprops=dict(arrowstyle="->",
+                                    color='green'))
+
+ax.flat[2].imshow(t1_unring - t1_gibbs, cmap="gray", vmin=0, vmax=10)
+ax.flat[2].annotate('Rings', fontsize=10, xy=(81, 70),
+                    color='red',
+                    xycoords='data', xytext=(30, 0),
+                    textcoords='offset points',
+                    arrowprops=dict(arrowstyle="->",
+                                    color='red'))
+ax.flat[2].annotate('Rings', fontsize=10, xy=(75, 50),
+                    color='red',
+                    xycoords='data', xytext=(30, 0),
+                    textcoords='offset points',
+                    arrowprops=dict(arrowstyle="->",
+                                    color='red'))
+plt.show()
+fig1.savefig('Gibbs_suppression_structural.png')
+
+"""
+.. figure:: Gibbs_suppression_structural.png
+   :align: center
+
+   Uncorrected and corrected structural images are shown in the left
+   and middle panels, while the difference between these images is shown
+   in the right panel.
+
+The image artificially corrupted with Gibb's artefacts is shown in the left
+panel. In this panels, the characteristic ringing profile of Gibbs artefacts
+can be visually appreciated (see intensity oscillations pointed by the red
+arrows). The corrected image is shown in the middle panel. One can appreciate
+that artefactual oscillations are visually suppressed without compromising
+the contrast between white and grey matter (e.g. details pointed by the green
+arrow). The difference between uncorrected and corrected data is plotted in
+the right panel which highlights the suppressed Gibbs ringing profile.
+
+
+Now let's show how to use the Gibbs suppression algorithm in diffusion-weighted
+images. In this way, we fetch the multi-shell diffusion-weighted dataset which
+was kindly provided by Hansen and Jespersen [5]_:
+
+"""
+
+from dipy.data import read_cenir_multib
+
+bvals = [200, 400, 1000, 2000]
+
+img, gtab = read_cenir_multib(bvals)
```

Comment:
I don't think this is the Hansen and Jespersen dataset! That one is `cfin_multib`. This one was provided by Romain Valabregue from CENIR.